package.json

```diff
@@ -43,7 +43,8 @@
     "tz-lookup": "^6.1.8",
     "winston": "^2.4.4",
     "winston-papertrail": "^1.0.2",
-    "yargs": "^12.0.1"
+    "yargs": "^12.0.1",
+    "node-fetch": "~2.2.x"
```

Comment:
Please drop this requirement as I think it's no longer needed. Also please remove your local changes to package-lock since I think they're also is not needed.

Reply:
Hm, think this was already removed in last push. Double-checked now, should be gone. Will try push new version this weekend.